python/arcticdb/arctic.py

```diff
@@ -138,6 +145,14 @@ def __init__(self, uri: str, encoding_version: EncodingVersion = DEFAULT_ENCODIN
 
                 Example connection strings are `lmdb:///home/user/my_lmdb` or `lmdb:///home/user/my_lmdb?map_size=2GB`.
 
+
+            In-Memory
+            ----
+                The in-memory connection scheme has the form ``"mem://"`.
+
+                The storage is local to the returned `Arctic` instance.
```

Comment:
Double backticks around `Arctic`?

Reply:
Thanks. Changes are in the docs PR here: https://github.com/man-group/ArcticDB/pull/895/files#diff-a9727c85029e3b2014647019b25f48df66c2670da1d17f0e2f7e279242f0a8a6R147-R149